llvm/lib/Target/AIE/CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ add_llvm_target(AIECodeGen
    Core
    CodeGen
    CodeGenTypes
+   IPO
```

Comment:
Check: this is the library where the internalize pass lives?

Reply:
Yes, and GlobalDCE.